src/runtime_src/core/pcie/tools/xbmgmt/xspi.cpp

```diff
@@ -879,6 +879,36 @@ bool XSPI_Flasher::bulkErase()
     return waitTxEmpty();
 }
 
+// Erases the entire flash
+bool XSPI_Flasher::fullErase() {
+    uint32_t numFlash = isDualQSPI(mDev.get()) ? 2 : 1;
```

Comment:
Since this variable will not change, it should be a const.  For example:
`const uint32_t numFlash = isDualQSPI(mDev.get()) ? 2 : 1;`